.eslintrc.js

```diff
@@ -33,5 +33,11 @@ module.exports = {
         'import/no-anonymous-default-export': 'off',
       },
     },
+    {
+      files: ['*.test.ts', '*.spec.ts', '*.test.tsx', '*.spec.tsx'],
+      rules: {
+        'lingui/no-unlocalized-strings': 'off',
```

Comment:
I don't think we should turn this off, this is a good way to identify strings that should be localized, in particular in the middle of the migration to use lingui

Reply:
This only in the test files, do we need the rule in them?

Reply:
ok nevermind then